client/src/utils/components/List.js

```diff
@@ -20,10 +20,22 @@ import React, { Fragment } from "react";
 import { Link } from "react-router-dom";
 import { Col } from "reactstrap";
 import Masonry from "react-masonry-css";
+import { faGlobe, faLock, faUserFriends } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
 import { TimeCaption } from "./TimeCaption";
 import { Pagination } from "./Pagination";
 import { ProjectTagList } from "../../project/shared";
 
+const VisibilityIcon = ({ visibility }) => {
+  const icon = {
+    public: <FontAwesomeIcon icon={faGlobe} />,
+    private: <FontAwesomeIcon icon={faLock} />,
+    internal: <FontAwesomeIcon icon={faUserFriends} />
+  };
+  return <span className="fs-small text-rk-text" title={visibility}>{ icon[visibility] || "" }</span>;
+};
```

Comment:
The `fs-small` class seems to make the icons *very* small. I'm not sure if it doesn't consider the relative size of the text, or maybe it's because the text is later placed in a `<div>` instead of a text element like `<h4>` or `<p>`.

Anyway, we could allow personalizing the size using the standard sizing utilities instead of the custom class `fs-small`

```suggestion
const VisibilityIcon = ({ visibility, size = "xs" }) => {
  const icon = {
    public: <FontAwesomeIcon icon={faGlobe} size={size} />,
    private: <FontAwesomeIcon icon={faLock} size={size} />,
    internal: <FontAwesomeIcon icon={faUserFriends} size={size} />
  };
  return <span className="text-rk-text" title={visibility}>{ icon[visibility] || "" }</span>;
};
```

Reply:
P.S. give the code a try first, I'm not 100% it works correctly :grin: